Source/WebKit/UIProcess/WebProcessCache.cpp

```diff
@@ -333,7 +334,8 @@ Ref<WebProcessProxy> WebProcessCache::CachedProcess::takeProcess()
 void WebProcessCache::CachedProcess::evictionTimerFired()
 {
     ASSERT(m_process);
-    m_process->processPool().webProcessCache().removeProcess(*m_process, ShouldShutDownProcess::Yes);
+    auto process = m_process.copyRef();
+    process->processPool().webProcessCache().removeProcess(*process, ShouldShutDownProcess::Yes);
```

Comment:
`checkedWebProcessCache()->` (already exists).

Reply:
Not sure what I was thinking here. Clearly, it doesn't make much difference to store m_process in RefPtr. It's WebProcessCache that needs to be stored in CheckedRef/Ref instead. Will fix.